server/src/server.ts

```diff
@@ -103,6 +103,18 @@ async function validateTextDocument(textDocument: TextDocument): Promise<void> {
   })
 
   const diagnostics: Diagnostic[] = []
+  if (
+    res.some(
+      (err) =>
+        err.text === "Field declarations don't require a `:`." ||
+        err.text ===
+          'Model declarations have to be indicated with the `model` keyword.',
```

Comment:
Maybe add a check for `type` here again? Otherwise also a typo `mofel` could trigger this, right?

Reply:
A typo could not trigger this, in that case a linter error would be displayed.